Sources/Confidence/FlagEvaluation.swift

```diff
@@ -59,6 +55,9 @@ extension FlagResolution {
             }
 
             guard let value = resolvedFlag.value else {
+                Task {
+                    await flagApplier?.apply(flagName: parsedKey.flag, resolveToken: self.resolveToken)
+                }
```

Comment:
Repeating this code is not ideal, but I believe it's less error prone to be explicit in which instances we emit apply. This would have probably prevented the bug in the first place

Reply:
I'm completely on board with that approach!

Reply:
But I'm a bit confused, should we send apply when we don't get any assigned value?

Reply:
I checked our backend and I think archived flags are never returned to the clients, so the `archived` case in here could be ignored